### DISCUSSION TITLE
MXnet will not received updates anymore, what will happen to GluonTS?

### DISCUSSION THREAD

**Discussion by MarcBresson** [awslabs/gluonts#3088] [General]
Hello, since september 2023, MXnet 'moved into the attic' (https://attic.apache.org/projects/mxnet.html). How will the development of GluonTS move onward ?

I saw in https://github.com/awslabs/gluonts/blob/dev/requirements/requirements-mxnet.txt that the version of mxnet used is 1.7 (https://github.com/apache/mxnet/releases/tag/1.7.0). However, when I install gluonts with `pip install "gluonts[mxnet]"`, mxnet is not downloaded (and rightfully because it isn't compatible with python 3.7 and newer). Could you bring context on the role mxnet really plays please ?

Thank you for your time

**Comment by abdulfatir** [comment]
@MarcBresson GluonTS currently supports two backends: `torch` and `mxnet`. Many models are available in torch. Check it out: https://github.com/awslabs/gluonts/tree/dev/src/gluonts/torch/model

**Reply by MarcBresson** [reply]
thank you for your reply.

How will GluonTS will phase out MXnet considering that it has very old dependencies (such as numpy 1.17 which is more than 3 years old). Will you try to switch the remaining MXnet-exclusive models to torch?
